google_guest_agent/non_windows_accounts.go

```diff
@@ -111,6 +111,13 @@ func (a *accountsMgr) set() error {
 	if sshKeys == nil {
 		logger.Debugf("initialize sshKeys map")
 		sshKeys = make(map[string][]string)
+		// only need to write to SSHD once in program execution
+		if err := writeSSHDConfigFile(); err != nil {
+			logger.Errorf("Error updating SSH config: %v.", err)
+		}
+		if err := systemctlStart("sshd"); err != nil {
```

Comment:
it would be `restart`, not `start`. but don't we already restart sshd?

Reply:
removed